src/amo/reducers/home.js

```diff
@@ -251,9 +265,16 @@ const reducer = (
     case SET_CLIENT_APP:
       return initialState;
 
+    case ABORT_FETCH_HOME_DATA:
+      return {
+        ...state,
+        isLoading: false,
+      };
+
     case FETCH_HOME_DATA:
       return {
         ...state,
+        isLoading: true,
```

Comment:
@kumar303 There is this test: https://github.com/mozilla/addons-frontend/blob/d9d68a0b19f376340b60be940039debc3542506f/tests/unit/amo/reducers/test_home.js#L214-L227

Rather than adding a separate test just to check the value of `isLoading`, I updated an existing test to verify that both `isLoading` and `resultsLoaded` are correct after a fetch. This seems okay to me because it represents the expected state for a a single action.

Do you feel it's important to have separate tests for both the `resultsLoaded` and the `isLoading` flags, or did you perhaps miss noticing this change in your review?

Reply:
@kumar303 I'm just waiting on an answer to this question before merging this patch. Thanks!

Reply:
Ship it!